### PR TITLE
Optional Quantization

### PR DESCRIPTION
**Pull request by kozlov721**
- Offline graph preparation is performed even with disabled quantization
- Added tests for rvc3 and rvc4 without quantization

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -59,14 +63,26 @@
...
+                "onnx",
+                "rvc4.disable_calibration true rvc3.disable_calibration true",
             ),
         ]
         for service in [
             "rvc2",
             "rvc2_superblob",
             "rvc3",
             "rvc3_quant",
+            "rvc3_non_quant",
```
Comment (tersekmatija) [inline tests/conftest.py:83]: Is "rvc3" quant or non-quant?

File: tests/conftest.py [hunk 2]
```diff
@@ -159,9 +189,9 @@ def prepare(
     else:
         file_url = f"s3://luxonis-test-bucket/modelconverter-test/archives/{model_name}.tar.xz"
 
-    if "quant" not in service:
+    if "quant" not in service or "non_quant" in service:
```
Comment (tersekmatija) [inline tests/conftest.py:192]: Can we make this more clear by just using "quant" when we want the quantize flow?

If in some case quant is only option, then we would just defined service with "_quant" in it?

**Review by tersekmatija — APPROVED** [review]

File: tests/conftest.py (hunk 1 above)
Comment (kozlov721) [reply tests/conftest.py:83]: "rvc3" is normal conversion but tested on the non-quant model. "rvc3_non_quant" is where calibration is explicitly disabled.

File: tests/conftest.py (hunk 2 above)
Comment (kozlov721) [reply tests/conftest.py:192]: Yeah, the tests would use some general refactoring eventually, they're quite convoluted. I'd probably do it all in some separate PR.

**Pull request merged**